crates/gateway-types/src/reply.rs

```diff
@@ -396,7 +396,7 @@ pub(crate) mod transaction {
                 builtin_instance_counter: value.builtin_instance_counter.into(),
                 n_steps: value.n_steps,
                 n_memory_holes: value.n_memory_holes,
-                data_availability: value.data_availability.map(Into::into),
+                data_availability: value.data_availability.unwrap_or_default().into(),
```

Comment:
Should the gateway types also be become non-optional?

Reply:
Isn't that covered by issue #1777?

Reply:
Good question, let's wait until 0.13.1 hits mainnet to see if we can. Integration seems to show that this is not possible rn (but that's integration).